measures/advanced_import_gbxml/resources/os_lib_adv_import.rb

```diff
@@ -819,92 +820,94 @@ def self.import_schs(runner, model, building_type, schedules, week_schedules, da
-        days = BuildingTypeHelper.create_prefix_day_array(building_type)
-
-        prefix_array = [day_type, date_range, days]
-        rules << prefix_array + time_value_array
-      end
-
-      if BuildingTypeHelper.is_on_6_days(building_type)
-        prefix_array = ["Sun", "1/1-12/31", "Sun"]
-        time_value_array = [[7, 0.0], [9, 0.1], [13, 0.2], [16, 0.1], [24, 0.0]]
-        rules << prefix_array + time_value_array
-      elsif BuildingTypeHelper.is_on_5_days(building_type)
-        prefix_array = ["Sun", "1/1-12/31", "Sun"]
-        time_value_array = [[24, 0.0]]
-        rules << prefix_array + time_value_array
-
-        prefix_array = ["Sat", "1/1-12/31", "Sat"]
-        time_value_array = [[7, 0.0], [9, 0.1], [13, 0.2], [16, 0.1], [24, 0.0]]
-        rules << prefix_array + time_value_array
-      end
-
-      if building_type == "SchoolOrUniversity"
-        prefix_array = ["January Break", "1/1-1/5", "Sun/Mon/Tue/Wed/Thu/Fri/Sat"]
-        time_value_array = [[24, 0.0]]
-        rules << prefix_array + time_value_array
-
-        prefix_array = ["Spring Break", "4/4-4/13", "Sun/Mon/Tue/Wed/Thu/Fri/Sat"]
-        time_value_array = [[24, 0.0]]
-        rules << prefix_array + time_value_array
 
-        prefix_array = ["Summer Break", "6/9-8/24", "Sun/Mon/Tue/Wed/Thu/Fri/Sat"]
-        time_value_array = [[24, 0.0]]
-        rules << prefix_array + time_value_array
+          # clean up excess values
+          time_value_array = []
+          last_val = nil
+          time_value_array_raw.reverse.each do |time_val|
+            if last_val.nil?
+              time_value_array << time_val
+            elsif last_val != time_val[1]
+              time_value_array << time_val
+            end
+            last_val = time_val[1]
+          end
+          time_value_array = time_value_array.reverse
+
+          # parse dayType attribute
+          case day_type
+          when 'HeatingDesignDay'
+            option['winter_design_day'].nil? ? options['winter_design_day'] = time_value_array : runner.registerWarning("Winter design day already defined for Schedule #{ruleset_name}")
+            # days = "HDD"
+          when 'CoolingDesignDay'
+            option['cooling_design_day'].nil? ? options['summer_design_day'] = time_value_array : runner.registerWarning("Summer design day already defined for Schedule #{ruleset_name}")
+            days = "CDD"
+          when 'Holiday'
+            options['holiday_day'].nil? ? options['holiday_day'] = time_value_array : runner.registerWarning("Holiday day already defined for Schedule #{ruleset_name}")
+          when 'Weekday'
+            days = 'Mon/Tue/Wed/Thu/Fri'
+          when 'Weekend'
+            days = 'Sat/Sun'
+          when 'WeekendOrHoliday'
+            days = 'Sat/Sun/Hol'
+          when 'Custom'
+            days = 'Custom' # TODO
+          when 'All' 
+            days = 'Mon/Tue/Wed/Thu/Fri/Sat/Sun'
+          else
+            days = day_type
+          end
```

Comment:
This section replaces the hard-coded day types applied by the BuildingTypeHelper class in /resources/building_type_helper.rb. Not sure if that helper was needed for special schedule parsing, can re-enable if so. Leaving the resources file in the measure for now.